api-server/rhelai-install.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# -*- indent-tabs-mode: nil; tab-width: 2; sh-indentation: 2; -*-
+
+# Install script for the API server
+## dependencies: go, git
+## required ENV vars: TAXONOMY_PATH, BASE_DIR
+
+set -x
+set -e
+set -o pipefail
+
+### installations
+
+rpm-ostree install gcc glibc-devel
```

Comment:
Pop a `sudo` in front of the rpm-ostree.

```
sudo rpm-ostree install gcc glibc-devel
```

 Can we test if those are already installed? rpm-ostree will throw an exit 1 if the package is already installed.